speech/google/cloud/speech/_gax.py

```diff
@@ -23,13 +29,23 @@
 from google.cloud.grpc.speech.v1beta1.cloud_speech_pb2 import (
     StreamingRecognizeRequest)
 
-
 from google.cloud.speech.alternative import Alternative
+from google.cloud._helpers import make_secure_stub
+from google.cloud.connection import DEFAULT_USER_AGENT
+from google.cloud.speech.operation import Operation
+from google.cloud.operation import register_type
```

Comment:
Put the non-`google.cloud.speech` imports above (and put them in alphabetical order)